### ISSUE TITLE
Can't export files with the Banner plugin

### ISSUE THREAD

**Issue by benoitbm** [KosmosisDire/obsidian-webpage-export#216]
Hello!
First of all, thank you for the plugin, it helps me sharing my D&D notes easily to my group, and it's genuinely easy to use and update (and the last update seems to help with updating and pushing the latest changes 👌).

However currently, I notice with the latest version (1.7.2 + Obsidian 1.4.12 on W10 Pro) that the export of some of my files fail, more precisely the files having a Banner header. It used to work previously, however, I notice that the plugin is not found in the "Plugin CSS" section.

![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/17157824/80a266dc-8032-435c-981a-424c68a41b92)

Each file with Banner has this error:

##### [Error] Could not export file: Jonni.md
```
TypeError: Cannot read properties of null (reading 'addEventListener')
```

##### [Error] Could not export file: Chef Cook.md
```
TypeError: Cannot read properties of null (reading 'addEventListener')
```

I've doubled check, it used to be detected and work with the 1.6 version, and Banners didn't got updated since.
Do you mind having a look at it please?
Thank you in advance and thanks again for this awesome plugin! 🙏

**Comment by KosmosisDire** [comment]
This is strange since my test vault is using the banners plugin to make sure that it works with the banners plugin. The plugin has been hiden from the CSS section as I have added my own css for the banners plugin because the css provided with the banners plugin was not functioning properly on the exported site. I will certainly look into this though.

**Comment by benoitbm** [comment]
On my side, if I disable the Banners plugin, it doesn't have the errors anymore (disabling, not removing nor changing the metadata).
I removed and installed again the plugin just in case, but it didn't change the outcome.

**Comment by benoitbm** [comment]
Actually, I found something interesting for the repro steps.
If I do a full export, I will have errors on these files
![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/17157824/1cf20db2-3f42-46b1-abc7-adba60173e5e)
However, if later, I enable this setting, and edit the pages which couldn't be exported, they export as expected, and with the banner.
It happens only on the first generation of these files, as after refreshing them bypass this issue.

**Comment by KosmosisDire** [comment]
Interesting thanks for the info, I'll look into it

**Comment by benoitbm** [comment]
Ok I noticed a pattern actually, the first Banner file can be exported without any issue, but the subsequent files are failing.
I thought first it was because of the metadata, I copy pasted the metadata of the working file to the next files, but they keep failing no matter the metadata.

**Comment by KosmosisDire** [comment]
Hmm very curious, could you actually open the console and take a screenshot of the errors there instead of copying them from the export log? You can open it with (Ctrl+Shift+I) and then click on "console" at the top

**Comment by benoitbm** [comment]
Sure thing!
![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/17157824/f43e53fe-a9a1-4ef0-8720-0c4fa3a69790)

```
p.js:1 TypeError: Cannot read properties of null (reading 'getBoundingClientRect')
    at Jdt (mermaid.min.js:102:73652)
    at mermaid.min.js:114:7250
    at Array.forEach (<anonymous>)
    at oBt (mermaid.min.js:114:6848)
    at Object.draw (mermaid.min.js:115:1386)
    at Object.XAe [as render] (mermaid.min.js:86:855)
(anonymous) @ app.js:1
plugin:webpage-html-export:56719 Could not export file: Jonni.md: 
 TypeError: Cannot read properties of null (reading 'addEventListener')
    at eval (plugin:obsidian-banners:28:861)
    at Array.forEach (<anonymous>)
    at OM (plugin:obsidian-banners:28:825)
    at Ge.onload (plugin:obsidian-banners:28:1402)
    at e.load (app.js:1:720327)
    at e.addChild (app.js:1:720728)
    at Object.addChild (app.js:1:1234913)
    at eval (plugin:obsidian-banners:28:3134)
    at t.postProcess (app.js:1:1235970)
    at t.postProcess (app.js:1:1234798)
    at renderMarkdownView (plugin:webpage-html-export:56188:21)
    at async renderMarkdownFile (plugin:webpage-html-export:56275:12)
    at async Object.renderFile (plugin:webpage-html-export:56126:14)
    at async Webpage.getDocumentHTML (plugin:webpage-html-export:55380:21)
    at async Webpage.create (plugin:webpage-html-export:55418:10)
    at async Website.createWithFiles (plugin:webpage-html-export:55704:16)
    at async HTMLExporter.exportFiles (plugin:webpage-html-export:57250:19)
    at async HTMLDivElement.eval (plugin:webpage-html-export:57366:21)
error @ plugin:webpage-html-export:56719
plugin:webpage-html-export:56719 Could not export file: Chef Cook.md: 
 TypeError: Cannot read properties of null (reading 'addEventListener')
    at eval (plugin:obsidian-banners:28:861)
    at Array.forEach (<anonymous>)
    at OM (plugin:obsidian-banners:28:825)
    at Ge.onload (plugin:obsidian-banners:28:1402)
    at e.load (app.js:1:720327)
    at e.addChild (app.js:1:720728)
    at Object.addChild (app.js:1:1234913)
    at eval (plugin:obsidian-banners:28:3134)
    at t.postProcess (app.js:1:1235970)
    at t.postProcess (app.js:1:1234798)
    at renderMarkdownView (plugin:webpage-html-export:56188:21)
    at async renderMarkdownFile (plugin:webpage-html-export:56275:12)
    at async Object.renderFile (plugin:webpage-html-export:56126:14)
    at async Webpage.getDocumentHTML (plugin:webpage-html-export:55380:21)
    at async Webpage.create (plugin:webpage-html-export:55418:10)
    at async Website.createWithFiles (plugin:webpage-html-export:55704:16)
    at async HTMLExporter.exportFiles (plugin:webpage-html-export:57250:19)
    at async HTMLDivElement.eval (plugin:webpage-html-export:57366:21)
e
```

**Comment by KosmosisDire** [comment]
This should be fixed in 1.8.0. Let me know if it still does not work as expected and reopen the issue. If you find any new issues please submit a new issue :)
Thanks!
